From ccb0230fe30737728a4d17054d18852dfa7a3b27 Mon Sep 17 00:00:00 2001
From: sajcho <saux.aarch64@gmail.com>
Date: Wed, 8 Feb 2023 10:57:32 +0100
Subject: [PATCH] Makefile.in: add support zstd

---
 Makefile.in | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index bce64e7..405e9f1 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -491,6 +491,10 @@ dist-xz: distdir
 	tardir=$(distdir) && $(am__tar) | XZ_OPT=$${XZ_OPT--e} xz -c >$(distdir).tar.xz
 	$(am__post_remove_distdir)
 
+dist-zstd: distdir
+	tardir=$(distdir) && $(am__tar) | zstd -c  >$(distdir).tar.zst
+	$(am__post_remove_distdir)
+
 dist-tarZ: distdir
 	tardir=$(distdir) && $(am__tar) | compress -c >$(distdir).tar.Z
 	$(am__post_remove_distdir)
@@ -521,6 +525,8 @@ distcheck: dist
 	  lzip -dc $(distdir).tar.lz | $(am__untar) ;;\
 	*.tar.xz*) \
 	  xz -dc $(distdir).tar.xz | $(am__untar) ;;\
+	*.tar.zst*) \
+	  zstd -dc $(distdir).tar.zst | $(am__untar) ;;\
 	*.tar.Z*) \
 	  uncompress -c $(distdir).tar.Z | $(am__untar) ;;\
 	*.shar.gz*) \
@@ -704,7 +710,7 @@ uninstall-am:
 .PHONY: $(RECURSIVE_CLEAN_TARGETS) $(RECURSIVE_TARGETS) CTAGS GTAGS \
 	all all-am am--refresh check check-am clean clean-cscope \
 	clean-generic cscope cscopelist cscopelist-recursive ctags \
-	ctags-recursive dist dist-all dist-bzip2 dist-gzip dist-lzip \
+	ctags-recursive dist dist-all dist-bzip2 dist-gzip dist-lzip dist-zstd \
 	dist-shar dist-tarZ dist-xz dist-zip distcheck distclean \
 	distclean-generic distclean-hdr distclean-tags distcleancheck \
 	distdir distuninstallcheck dvi dvi-am html html-am info \
-- 
2.39.1

From 2f8e2f00fa67d7e1c9bc95ae550dd4f5c342acc5 Mon Sep 17 00:00:00 2001
From: sajcho <saux.aarch64@gmail.com>
Date: Wed, 8 Feb 2023 11:01:26 +0100
Subject: [PATCH] installtransaction.cpp: define the architecture of the
 created package

---
 src/installtransaction.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/installtransaction.cpp b/src/installtransaction.cpp
index 59834f1..11eb9bb 100644
--- a/src/installtransaction.cpp
+++ b/src/installtransaction.cpp
@@ -371,7 +371,7 @@ InstallTransaction::installPackage( const Package* package,
             args +=
                 package->name()    + "#" +
                 package->version() + "-" +
-                package->release() + ".pkg.tar." + getPkgmkCompressionMode();
+                package->release() + ".$(uname -m).pkg.tar." + getPkgmkCompressionMode();
 
 
             // - inform the user about what's happening
-- 
2.39.1

From 0c13aa0ed84b6a5b619860a40480b903706f11da Mon Sep 17 00:00:00 2001
From: sajcho <saux.aarch64@gmail.com>
Date: Wed, 8 Feb 2023 11:22:45 +0100
Subject: [PATCH] remove blank spaces

---
 doc/Makefile.am            |   2 +-
 doc/Makefile.in            |   2 +-
 doc/prt-cache.8            |  14 +--
 doc/prt-get.8              | 182 ++++++++++++++++++-------------------
 doc/prt-get.conf.5         |  10 +-
 misc/prt-get.deplist       |  46 +++++-----
 misc/prt-get_complete      |   4 +-
 src/Makefile.in            |   2 +-
 src/argparser.h            |   2 +-
 src/datafileparser.cpp     |  14 +--
 src/datafileparser.h       |  10 +-
 src/depresolver.cpp        |   2 +-
 src/depresolver.h          |   2 +-
 src/file.cpp               |   8 +-
 src/installtransaction.cpp |   2 +-
 src/installtransaction.h   |   2 +-
 src/locker.h               |   2 +-
 src/lockfile.cpp           |  30 +++---
 src/lockfile.h             |  20 ++--
 src/package.cpp            |   6 +-
 src/package.h              |   4 +-
 src/pg_regex.cpp           |  22 ++---
 src/pg_regex.h             |  10 +-
 src/pkgdb.cpp              |   2 +-
 src/process.h              |   8 +-
 src/prtget.cpp             |  16 ++--
 src/signaldispatcher.cpp   |   2 +-
 src/signaldispatcher.h     |  26 +++---
 src/stringhelper.cpp       |   2 +-
 src/versioncomparator.cpp  |   2 +-
 src/versioncomparator.h    |   8 +-
 31 files changed, 232 insertions(+), 232 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 14e4127..a565d5c 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -2,7 +2,7 @@
 man_MANS = prt-get.8 prt-cache.8 prt-get.conf.5
 
 clean-local:
-	rm -rf *.log *.aux *.toc *.dvi *.idx *.pdf api 
+	rm -rf *.log *.aux *.toc *.dvi *.idx *.pdf api
 
 apidoc:
 	doxygen
diff --git a/doc/Makefile.in b/doc/Makefile.in
index bb3e743..3ccc27e 100644
--- a/doc/Makefile.in
+++ b/doc/Makefile.in
@@ -482,7 +482,7 @@ uninstall-man: uninstall-man5 uninstall-man8
 
 
 clean-local:
-	rm -rf *.log *.aux *.toc *.dvi *.idx *.pdf api 
+	rm -rf *.log *.aux *.toc *.dvi *.idx *.pdf api
 
 apidoc:
 	doxygen
diff --git a/doc/prt-cache.8 b/doc/prt-cache.8
index 9144119..c4e1bef 100644
--- a/doc/prt-cache.8
+++ b/doc/prt-cache.8
@@ -4,11 +4,11 @@
 .PU
 .TH prt-cache 8
 .SH "NAME"
-.LP 
+.LP
 prt\-cache \- call prt\-get using a cache file as a data source
 .SH "SYNOPSIS"
 .B prt\-cache command [options] <arguments>
-.br 
+.br
 .SH "DESCRIPTION"
 prt\-cache is a synonym for
 .B prt\-get --cache.
@@ -19,23 +19,23 @@ for details on how to use prt-get
 .PP
 Even though there are almost restrictions on using prt-cache over prt-get,
 it's mostly recommended for the following commands:
-.B search, dsearch, list, info, depends, quickdep, diff, quickdiff 
-and 
+.B search, dsearch, list, info, depends, quickdep, diff, quickdiff
+and
 .B info.
 It won't work for the
 .B dup
 command.
 
 .PP
-Using 
-.B prt-cache isinst 
+Using
+.B prt-cache isinst
 is not recommended (really not!)
 
 .PP
 Note that if the cache is not up to date (and there's a newer port in the
 ports tree),
 .B prt-cache install
-and 
+and
 .B prt-cache update
 will use the version from the ports tree anyway, so there's no real
 risk.
diff --git a/doc/prt-get.8 b/doc/prt-get.8
index bc1938d..4f7b348 100644
--- a/doc/prt-get.8
+++ b/doc/prt-get.8
@@ -1,17 +1,17 @@
 .\" man page for prt-get
 .\" Johannes Winkelmann, jw@tks6.net
-.\" 
+.\"
 .\" my first man page, so forgive me for the errors :-)
-.\" 
+.\"
 .\" .PU
 .TH "prt-get" "8" "" "" ""
 .SH "NAME"
-.LP 
+.LP
 prt\-get \- an advanced package management tool to be used with
 pkgutils from CRUX (see http://www.crux.nu)
 .SH "SYNOPSIS"
 .B prt\-get command [options] <arguments>
-.br 
+.br
 .SH "DESCRIPTION"
 prt\-get is a package management tool which provides additional
 functionality to crux' package management system. It works with the
@@ -19,27 +19,27 @@ local ports tree and is therefore fully compatible with ports(8) and
 pkgmk(8)/pkgadd(8). It offers the following features:
 
 .PP
-.TP 
+.TP
 \ \ \ \(bu
 abstract ports installation/update from file system
 
-.TP 
+.TP
 \ \ \ \(bu
 install/update a list of packages with one command
 
-.TP 
+.TP
 \ \ \ \(bu
 list dependencies for a list of packages
 
-.TP 
+.TP
 \ \ \ \(bu
 show information about ports
 
-.TP 
+.TP
 \ \ \ \(bu
 search within the ports
 
-.PP 
+.PP
 
 What prt\-get basically does is installing and upgrading packages, using
 pkgmk and pkgadd. Additionally, you don't have be in the port's
@@ -48,7 +48,7 @@ itself in a list of directories specified in /etc/prt\-get.conf. This
 allows you to just install or update a package, without caring where
 it actually is located on your file system.
 
-.PP 
+.PP
 
 prt\-get also offers some features like searching ports by name,
 showing information about ports (without installing them of course)
@@ -58,7 +58,7 @@ requirement for crux packages and therefore not always accurate. There's
 a possiblity to use an external dependency map to get dependency
 listings for ports which don't provide them in their Pkgfile.
 
-.PP 
+.PP
 
 prt\-get has a test mode so you can see what effect an install/update
 operation would have. Use the \-\-test switch for this (see also:
@@ -69,37 +69,37 @@ prt\-get returns 0 on success and a non-zero value otherwise (exact
 value -> meaning mapping to be determined...)
 
 .SH "COMMANDS"
-prt\-get uses so called commands, which always have to be the first 
+prt\-get uses so called commands, which always have to be the first
 non-option argument passed. This is very similar to
 .B cvs(1).
 [command] can be one of the following:
 
 
-.TP 
+.TP
 .B install [\-\-margs] [\-\-aargs] [\-\-log] <package1> [<package2> ...]
 install all packages in the listed order. Note that you can do this
 from any directory
 
-.TP 
+.TP
 .B depinst [\-\-margs] [\-\-aargs] [\-\-log] <package1> [<package2> ...]
 install all packages in the listed order including their dependencies.
 Note that outdated packages won't be updated.
 
-.TP 
+.TP
 .B grpinst [\-\-margs] [\-\-aargs]  [\-\-log] <package1> [<package2> ...]
 install all packages in the listed order, but stop if installation of
 one package fails. This can be used to install packages where package2
 depends on package1
 
-.TP 
+.TP
 .B update [\-\-margs] [\-\-aargs]  [\-\-log] <package1> [<package2> ...]
 update all packages listed in this order
 
-.TP 
+.TP
 .B remove <package1> [<package2> ...]
 remove packages listed in this order
 
-.TP 
+.TP
 .B sysup [\-\-nodeps]
 Update all installed packages which are outdated. Sorts by dependencies
 by default. If you don't want prt-get to resolve the dependencies, use
@@ -112,17 +112,17 @@ changes in the ports tree. If you want to update only diffs which have
 a lower version installed than the one in the ports tree, use the
 --prefer-higher option.
 
-.TP 
+.TP
 .B lock <package1> [<package2>...]
 Do not update these packages in a
 .B sysup
 operation
 
-.TP 
-.B unlock <package1> [<package2>...] 
+.TP
+.B unlock <package1> [<package2>...]
 Remove lock from these packages
 
-.TP 
+.TP
 .B listlocked [-v|-vv]
 List names of packages which are locked. As always,
 .B -v
@@ -132,7 +132,7 @@ also includes the description from the Pkgfile. Note that the
 .B -vv
 switch will slow down the operation remarkably.
 
-.TP 
+.TP
 .B diff [--all] [<package1> <package2> ...]
 show differences between installed packages and ports in the ports
 tree. If arguments are given, shows only differences for these
@@ -144,7 +144,7 @@ packages are only displayed if you use the --all switch. If you want
 to see only diffs which have a lower version installed than the one in
 the ports tree, use the --prefer-higher option.
 
-.TP 
+.TP
 .B quickdiff
 prints a simple list of packages which have a different version in the
 ports tree than what is installed. This is meant as an input for
@@ -153,13 +153,13 @@ so you can update all ports on your system with one simple command. If you want
 to see only diffs which have a lower version installed than the one in
 the ports tree, use the --prefer-higher option.
 
-.TP 
+.TP
 .B search [\-v|\-vv] [--path] [--regex]<expr>
 Search the ports tree for
 .B expr
 in their name
 
-.TP 
+.TP
 .B dsearch [\-v|\-vv] [--path]  [--regex]<expr>
 Search the ports tree for
 .B expr
@@ -169,7 +169,7 @@ makes it rather slow; if you like this, consider using the cache
 functionality, so you only have to spend this time once after updating
 the ports tree has been updated.
 
-.TP 
+.TP
 .B fsearch [--full] [--regex] <pattern>
 Search the ports tree for
 .B pattern
@@ -180,25 +180,25 @@ default. Pattern can be a shell-like wildcard pattern (e.g prt-get
 fsearch "*.h") or regexps.
 
 
-.TP 
+.TP
 .B info <port>
-Print available info for a port 
+Print available info for a port
 
-.TP 
+.TP
 .B path <port>
 Print the path of a port
 
-.TP 
+.TP
 .B readme <port>
 Print the port's README file if it exists; if set, uses $PAGER
 
-.TP 
+.TP
 .B depends <package1> [<package2> ...]
 print a recursive list of dependencies to install the packages passed
 as argument. It shows a list with ports which are found, plus a list
 of ports which are missing
 
-.TP 
+.TP
 .B quickdep <package1> [<package2> ...]
 print a simple list of recursive dependencies for all the ports passed
 as argument in a simple format to be used by
@@ -206,23 +206,23 @@ as argument in a simple format to be used by
 to make an install including dependencies
 .B Does not display dependencies which are not in the ports tree
 
-.TP 
+.TP
 .B dependent <package>
 print a list of package which depend on
 .B package.
- 
+
 Usually shows dependent packages which are installed. To see all dependencies,
 add the --all switch; use --recursive to get a recursive list (without
 duplication), and --tree to get a nicely indented one
 
-.TP 
+.TP
 .B deptree <package>
 print a tree of the dependencies of the package
-.B package. 
+.B package.
 Subtrees already shown are marked with '-->' to save some space, in
 order to show them all, add the --all switch
 
-.TP 
+.TP
 .B dup [-v] [format]
 List ports which can be found in multiple directories configured in
 .B /etc/prt-get.conf
@@ -230,19 +230,19 @@ Use the verbose switch to simulate the output of version 5.12 and older (likely
 to go away in the future). The format string can be used to create user
 specified formats. The following symbols are currently replaced:
 
-.TP 
+.TP
 \ \ \ \(bu
 %n \-> name of the port
 
-.TP 
+.TP
 \ \ \ \(bu
 %p1 \-> Full path (including name) to port taking precendence
 
-.TP 
+.TP
 \ \ \ \(bu
 %p2 \-> Full path (including name) to port being hidden
 
-.TP 
+.TP
 \ \ \ \(bu
 %v1 \-> Version of port taking precendence
 
@@ -251,7 +251,7 @@ specified formats. The following symbols are currently replaced:
 %v2 \-> Version of port being hidden
 
 
-.TP 
+.TP
 .B list [\-v|\-vv] [filter] [--path] [--regex]
 List ports available in the ports tree. It's basically the same as
 .B ports \-l
@@ -262,15 +262,15 @@ shell like
 .B wildcards
 for the list command. Make sure you escape where needed
 
-.TP 
+.TP
 .B printf <format string1> [\-\-sort=<format string2>] [\-\-filter=<filter>]
 Print formated port list format string can contain variables, which
 are replaced like this:
-.TP 
+.TP
 \ \ \ \(bu
 %n \-> name
 
-.TP 
+.TP
 \ \ \ \(bu
 %p \-> path
 
@@ -278,47 +278,47 @@ are replaced like this:
 \ \ \ \(bu
 %v \-> version
 
-.TP 
+.TP
 \ \ \ \(bu
 %r \-> release
 
-.TP 
+.TP
 \ \ \ \(bu
 %d \-> description
 
-.TP 
+.TP
 \ \ \ \(bu
 %e \-> dependencies
 
-.TP 
+.TP
 \ \ \ \(bu
 %u \-> URL
 
-.TP 
+.TP
 \ \ \ \(bu
 %P -> Packager
 
-.TP 
+.TP
 \ \ \ \(bu
 %M -> Maintainer
 
-.TP 
+.TP
 \ \ \ \(bu
 %R -> Readme ("yes"/"no")
 
-.TP 
+.TP
 \ \ \ \(bu
 %E -> pre-install script ("yes"/"no")
 
-.TP 
+.TP
 \ \ \ \(bu
 %O -> post-install script ("yes"/"no")
 
-.TP 
+.TP
 \ \ \ \(bu
 %l -> is locked ("yes"/"no")
 
-.TP 
+.TP
 \ \ \ \(bu
 %i \-> "no" if not installed, "yes" if it's installed and
 up to date and "diff" if it's installed and a new version is in the
@@ -331,7 +331,7 @@ as format string1 and is used to sort the output. You can specify a
 to filter by package name.
 
 
-.TP 
+.TP
 .B listinst [\-v|\-vv] [filter] [--regex] [--depsort]
 List installed ports. It's basically the same as
 .B pkginfo \-i,
@@ -347,17 +347,17 @@ for the listinst command. Make sure you escape where needed. Finally, by
 default it's sorted alphabetically; use the \-\-depsort switch to sort by
 dependencies
 
-.TP 
+.TP
 .B listorphans [\-v|\-vv]
 List installed ports which have no dependent packages
 
 
 
-.TP 
+.TP
 .B isinst <package>
 Check whether a package is installed. The same as
 .B pkginfo \-i|grep ^package
-.TP 
+.TP
 
 .TP
 .B current <package>
@@ -378,19 +378,19 @@ Edit the file using the editor specified in the $EDITOR environment variable.
 If <file> is not specified, 'Pkgfile' is used
 
 
-.TP 
+.TP
 .B help
 Shows a help screen
 
-.TP 
+.TP
 .B dumpconfig
-Dump the configuration to the current terminal 
+Dump the configuration to the current terminal
 
-.TP 
-.B version 
+.TP
+.B version
 Shows the current version of prt\-get
 
-.TP 
+.TP
 .B cache
 create a cache file from the ports tree to be used by prt\-get using the
 \-\-cache option. Remember to run prt\-get cache each time you update the
@@ -443,45 +443,45 @@ Keep working directory, implies 'pkgmk -kw'; same as --margs=-kw
 Don't install those packages, even if they're listed as dependency
 
 
-.TP 
+.TP
 .B \-\-cache
 Use cache file for this command
 
-.TP 
+.TP
 .B \-\-test
 Dry run, don't actually install anything, mostly interesting for
 .B prt\-get install, prt\-get grpinst, prt\-get update, prt\-get sysup
 
-.TP 
+.TP
 .B \-\-pre-install
 Execute pre-install script if it's there
 
-.TP 
+.TP
 .B \-\-post-install
 Execute post-install script if it's there
 
-.TP 
+.TP
 .B \-\-install-scripts
 Execute pre-install and post-install script if they're there
 
-.TP 
+.TP
 .B \-\-no-std-config
 Don't parse the default configuration file
 
-.TP 
+.TP
 .B \-\-config-prepend=<string>
 Prepend <string> to configuration
 
-.TP 
+.TP
 .B \-\-config-append=<string>
 Append <string> to configuration
 
-.TP 
+.TP
 .B \-\-config-set=<string>
 Set <string> in configuration, overriding configuration file
 
 
-.TP 
+.TP
 .B \-v, \-vv
 verbose and more verbose, relevant for
 .B prt\-get search, prt\-get list
@@ -496,18 +496,18 @@ Show path info; used in 'search', 'dsearch', 'list' and 'depends'
 .B \-\-regex
 Interpret filter and search pattern as regular expression
 
-.TP 
+.TP
 .B \-\-margs="...", e.g. \-\-margs="\-im"
 arguments to be passed to pkgmk, relevant for
 .B prt\-get install, prt\-get grpinst, prt\-get update
 note that \-d is already passed to pkgmk anyway
 
-.TP 
+.TP
 .B \-\-aargs="...", e.g. \-\-aargs="\-f"
 arguments to be passed to pkgadd, relevant for
 .B prt\-get install, prt\-get grpinst, prt\-get update
 
-.TP 
+.TP
 .B \-\-rargs="..."
 arguments to be passed to pkgrm; currently not used, implemented for
 future uses and consistency reasons
@@ -523,7 +523,7 @@ quickdiff and sysup.
 Override the 'preferhigher' configuration option
 
 
-.TP 
+.TP
 .B \-\-config=<file>
 Use alternative configuration file to read ports directories from
 
@@ -533,43 +533,43 @@ Use <dir> as root directory for your installation; allows to install to
 a different directory than '/'. In daily usage, this option is not
 required; helpful if you're developing an independent installation.
 
-.TP 
+.TP
 .B \-\-log
 Write build output to log file
 
 .SH "CONFIGURATION"
-.TP 
+.TP
 See man prt\-get.conf(5)
 
 .SH "EXAMPLES"
-.TP 
+.TP
 .B prt\-get install irssi
 download, build and install irssi, with one simple command
 
-.TP 
+.TP
 .B prt\-get install indent mplayer
 install indent and mplayer
 
-.TP 
+.TP
 .B prt\-get update openssh
 update your current version of openssh :\-)
 
-.TP 
+.TP
 .B prt\-get info kdelibs
 show info about kdelibs
 
-.TP 
+.TP
 .B prt\-get search icq
 Search all ports containing 'icq' in their name
 
-.TP 
+.TP
 .B prt\-get grpinst \-\-test `prt\-get quickdep kdenetwork`
 Show what would happen if you installed all packages needed for
 kdenetwork (test mode). Remove \-\-test to install kdenetwork and all
 it's dependencies. Remember that grpinst stops installing when one
 installing of one package fails
 
-.TP 
+.TP
 .B prt\-get update `prt\-get quickdiff`
 Update all packages where a different version is in the ports tree
 
diff --git a/doc/prt-get.conf.5 b/doc/prt-get.conf.5
index 7006277..e350831 100644
--- a/doc/prt-get.conf.5
+++ b/doc/prt-get.conf.5
@@ -5,7 +5,7 @@
 .PU
 .TH prt-get.conf 5
 .SH "NAME"
-.LP 
+.LP
 prt-get.conf - Configuration file for prt-get
 
 .SH "DESCRIPTION"
@@ -67,8 +67,8 @@ The valid keys are
 .B prtdir
 and
 .B cachefile.
-The value for 
-.B prtdir 
+The value for
+.B prtdir
 is a directory, the value for
 .B cachefile
 is a file to be used as cache
@@ -111,7 +111,7 @@ which can be set to 'enabled' or 'disabled'
 .B logmode
 which can be set to 'append' or 'overwrite'
 .B rmlog_on_success
-which can be set to 'yes' or 'no'; when set to yes, log files of 
+which can be set to 'yes' or 'no'; when set to yes, log files of
 successful builds will be removed
 .B logfile
 which represent the file path. All occurances of %n in logfile are
@@ -150,7 +150,7 @@ ports directory instead and use symlinks for the ports you want to use
 .LP
 You can write comments after a '#' character. If you have '#'
 characters in you pathes, there's no way to escape them (as there is no
-way to escape ':' characters). Complain to the author if this is a 
+way to escape ':' characters). Complain to the author if this is a
 problem :-)
 
 
diff --git a/misc/prt-get.deplist b/misc/prt-get.deplist
index 51145cb..749e3d1 100644
--- a/misc/prt-get.deplist
+++ b/misc/prt-get.deplist
@@ -1,28 +1,28 @@
 # dependencies for opt ports, 16 May 2004, Jürgen Daubert
 
-atk         : glib        
-audiofile   : pkgconfig   
-cvs         : openssl     
-emacs       : x11         libtiff     
-esound      : audiofile   
-fetchmail   : openssl     
-firefox     : gtk         libidl      
-fontconfig  : freetype    expat       pkgconfig   
-glib        : gettext     pkgconfig   
+atk         : glib
+audiofile   : pkgconfig
+cvs         : openssl
+emacs       : x11         libtiff
+esound      : audiofile
+fetchmail   : openssl
+firefox     : gtk         libidl
+fontconfig  : freetype    expat       pkgconfig
+glib        : gettext     pkgconfig
 gtk         : atk         pango       libtiff
-libidl      : glib        
-libogg      : pkgconfig   
-libpng      : pkgconfig   
-libtiff     : libjpeg     
-libvorbis   : libogg      
-openssh     : openssl     
-pango       : glib        x11         
-pine        : openssl     
-ppp         : openssl     
-webfs       : openssl     
-windowmaker : x11         libtiff     libungif    
-xchat       : gtk         openssl     
-x11         : fontconfig  libpng   
+libidl      : glib
+libogg      : pkgconfig
+libpng      : pkgconfig
+libtiff     : libjpeg
+libvorbis   : libogg
+openssh     : openssl
+pango       : glib        x11
+pine        : openssl
+ppp         : openssl
+webfs       : openssl
+windowmaker : x11         libtiff     libungif
+xchat       : gtk         openssl
+x11         : fontconfig  libpng
 zip         : unzip
 
-# End of file  
+# End of file
diff --git a/misc/prt-get_complete b/misc/prt-get_complete
index a06dfcd..fdeb84b 100644
--- a/misc/prt-get_complete
+++ b/misc/prt-get_complete
@@ -1,4 +1,4 @@
-# prt-get(8), prt-cache(8) completion by Johannes Winkelmann, jw@tks6.net 
+# prt-get(8), prt-cache(8) completion by Johannes Winkelmann, jw@tks6.net
 # problems: options ending on = should not add a space afterwards
 #
 # 07.07.2002:   use prt-get listinst (from 0.3.1pre1) instead of pkginfo
@@ -23,7 +23,7 @@ _prt-get()
                     remove listinst dumpconfig listorphans' $cur ))
         fi
 
-       
+
         if [ $COMP_CWORD '>' 1 ]; then
             if [[ "$cur" != -* ]]; then
 	        case ${COMP_WORDS[1]} in
diff --git a/src/Makefile.in b/src/Makefile.in
index b049d2a..a7411d5 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -283,7 +283,7 @@ uninstall-binPROGRAMS:
 
 clean-binPROGRAMS:
 	-test -z "$(bin_PROGRAMS)" || rm -f $(bin_PROGRAMS)
-prt-get$(EXEEXT): $(prt_get_OBJECTS) $(prt_get_DEPENDENCIES) $(EXTRA_prt_get_DEPENDENCIES) 
+prt-get$(EXEEXT): $(prt_get_OBJECTS) $(prt_get_DEPENDENCIES) $(EXTRA_prt_get_DEPENDENCIES)
 	@rm -f prt-get$(EXEEXT)
 	$(CXXLINK) $(prt_get_OBJECTS) $(prt_get_LDADD) $(LIBS)
 
diff --git a/src/argparser.h b/src/argparser.h
index c24e7bd..781ae68 100644
--- a/src/argparser.h
+++ b/src/argparser.h
@@ -112,7 +112,7 @@ private:
 
     bool m_recursive;
     bool m_printTree;
-    
+
     bool m_depSort;
 
     string m_alternateConfigFile;
diff --git a/src/datafileparser.cpp b/src/datafileparser.cpp
index 4291f26..268ee62 100644
--- a/src/datafileparser.cpp
+++ b/src/datafileparser.cpp
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2002 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 #include <cstdio>
@@ -17,17 +17,17 @@
 using namespace std;
 using namespace StringHelper;
 
-bool DataFileParser::parse(const string& fileName, 
+bool DataFileParser::parse(const string& fileName,
                            map<string, string>& target)
 {
     FILE* fd = fopen(fileName.c_str(), "r");
     char buffer[512];
     string input;
-    
+
     if (!fd) {
         return false;
     }
-    
+
     while (fgets(buffer, 512, fd)) {
         input = buffer;
         input = stripWhiteSpace(input);
diff --git a/src/datafileparser.h b/src/datafileparser.h
index bc1ca9a..31f8e8a 100644
--- a/src/datafileparser.h
+++ b/src/datafileparser.h
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2004 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 #ifndef _DATAFILEPARSER_H_
@@ -22,7 +22,7 @@
 class DataFileParser
 {
 public:
-    static bool parse(const std::string& fileName, 
+    static bool parse(const std::string& fileName,
                       std::map<std::string, std::string>& target);
 };
 
diff --git a/src/depresolver.cpp b/src/depresolver.cpp
index ca5d609..54870e4 100644
--- a/src/depresolver.cpp
+++ b/src/depresolver.cpp
@@ -30,7 +30,7 @@ void DepResolver::addDependency( int first, int second )
 
 /*!
   resolve the dependencies
-  \param result a list which will be filled with resulting indexes in 
+  \param result a list which will be filled with resulting indexes in
   the correct order
   \return true on success, false otherwise (cyclic dependencies)
 */
diff --git a/src/depresolver.h b/src/depresolver.h
index 7bbd89a..9d09077 100644
--- a/src/depresolver.h
+++ b/src/depresolver.h
@@ -18,7 +18,7 @@ using namespace std;
 /*!
   \class DepResolver
   \brief a dependency resolver
-  
+
   A dependency resolver
 */
 class DepResolver
diff --git a/src/file.cpp b/src/file.cpp
index 04f17fe..88432c4 100644
--- a/src/file.cpp
+++ b/src/file.cpp
@@ -49,7 +49,7 @@ bool grep( const string& fileName,
     char* end;
     string line;
     string entry;
-    
+
     RegEx re(pattern);
 
     while ( fgets( input, length, fp ) ) {
@@ -61,19 +61,19 @@ bool grep( const string& fileName,
             // prepend slash to string
             p--;
             p[0] = '/';
-            
+
             entry = p;
             end = strstr(p, "->");
             if (end) {
                 *end = '\0';
             }
             p[strlen(p)-1] = '\0'; // strip newline
-            
+
             char* name = p;
             if (!fullPath) {
                 name = basename(p);
             }
-                
+
             if (useRegex) {
                 if (re.match(name)) {
                     result.push_back(StringHelper::stripWhiteSpace(entry));
diff --git a/src/installtransaction.cpp b/src/installtransaction.cpp
index 11eb9bb..0e1f017 100644
--- a/src/installtransaction.cpp
+++ b/src/installtransaction.cpp
@@ -114,7 +114,7 @@ InstallTransaction::InstallTransaction( const string& name,
                                         PkgDB* pkgDB,
                                         const Configuration* config )
     : m_pkgDB( pkgDB ),
-      m_resolver(), 
+      m_resolver(),
       m_repo( repo ),
       m_depCalced( false ),
       m_installedPackages(),
diff --git a/src/installtransaction.h b/src/installtransaction.h
index 113abf2..9dbae79 100644
--- a/src/installtransaction.h
+++ b/src/installtransaction.h
@@ -111,7 +111,7 @@ private:
                                   InstallInfo& info ) const;
 
     static string getPkgmkSetting(const string& setting);
-    static string getPkgmkSettingFromFile(const string& setting, 
+    static string getPkgmkSettingFromFile(const string& setting,
                                           const string& fileName);
 
     PkgDB* m_pkgDB;
diff --git a/src/locker.h b/src/locker.h
index d7d5a9b..f67a9c4 100644
--- a/src/locker.h
+++ b/src/locker.h
@@ -24,7 +24,7 @@ using namespace std;
  * - marked in prt-get diff
  * - not shown in prt-get quickdiff
  * - not updated in prt-get sysup
- * 
+ *
  * remember to call store!
  */
 class Locker
diff --git a/src/lockfile.cpp b/src/lockfile.cpp
index 340691b..5285805 100644
--- a/src/lockfile.cpp
+++ b/src/lockfile.cpp
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2002 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 #include <sys/types.h>
@@ -29,8 +29,8 @@ string LockFile::READ_WRITE_LOCK_STRING  = "read_write_lock";
   \brief a file locking class
  */
 LockFile::LockFile()
-    : m_readLock( false ), 
-      m_writeLock( false )   
+    : m_readLock( false ),
+      m_writeLock( false )
 {
 }
 
@@ -59,10 +59,10 @@ bool LockFile::lockRead()
         }
     } else {
         // file exists, but it could be a read lock
-        
+
         // TODO: allow locking, increment lock count
     }
-    
+
     // couldn't lock
     return false;
 }
@@ -78,7 +78,7 @@ bool LockFile::lockWrite()
         // we already lock it, so it's a good lock
         return true;
     }
-    
+
     if ( m_readLock ) {
         // can't lock for read and write separatedly
         return false;
@@ -86,7 +86,7 @@ bool LockFile::lockWrite()
 
     struct stat buf;
     if ( stat( m_fileName.c_str(), &buf) == -1 ) {
-        // file can not be stat'ed 
+        // file can not be stat'ed
         FILE* fp = fopen( m_fileName.c_str(), "w" );
         if ( fp ) {
             // file did not exist, could be created
@@ -96,7 +96,7 @@ bool LockFile::lockWrite()
             return true;
         }
     }
-    
+
     // couldn't lock
     return false;
 }
@@ -107,7 +107,7 @@ bool LockFile::lockReadWrite()
         // we already lock it, so it's a good lock
         return true;
     }
-    
+
     if ( m_writeLock || m_readLock ) {
         // can't lock for read and write separatedly
         return false;
@@ -124,10 +124,10 @@ bool LockFile::lockReadWrite()
             return true;
         }
     }
-    
+
     // couldn't lock
     return false;
-    
+
 }
 
 bool LockFile::unlock()
@@ -139,7 +139,7 @@ bool LockFile::unlock()
             return true;
         }
     }
-    
+
     return false;
 }
 
diff --git a/src/lockfile.h b/src/lockfile.h
index fb1415f..9eebb38 100644
--- a/src/lockfile.h
+++ b/src/lockfile.h
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2002 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 #ifndef _LOCKFILE_H_
@@ -18,15 +18,15 @@ class LockFile
 {
 public:
     LockFile();
-    
+
     bool lockRead();
     bool lockWrite();
     bool lockReadWrite();
-    
+
     bool unlock();
-    
+
     void setFile( const std::string& fileName );
-    
+
 private:
     std::string m_fileName;
 
@@ -35,11 +35,11 @@ private:
     bool m_readWriteLock;
 
     static std::string LOCK_SUFFIX;
-    
+
     static std::string WRITE_LOCK_STRING;
     static std::string READ_LOCK_STRING;
     static std::string READ_WRITE_LOCK_STRING;
-    
+
 };
 
 #endif /* _LOCKFILE_H_ */
diff --git a/src/package.cpp b/src/package.cpp
index 458451c..4e6a36c 100644
--- a/src/package.cpp
+++ b/src/package.cpp
@@ -315,10 +315,10 @@ void Package::expandShellCommands(std::string& input,
 
     for (int i = 0; i < TAG_COUNT; ++i) {
         string::size_type pos, dpos = 0;
-        
+
         pos = 0;
         while ((pos = input.find(startTag[i], pos)) != string::npos) {
-            
+
             if (unameBuf.release) {
                 input = replaceAll(input,
                                    startTag[i] + "uname -r" + endTag[i],
@@ -345,7 +345,7 @@ void Package::expandShellCommands(std::string& input,
                 input = input.substr(0, dpos) + timeBuf +
                     input.substr(endpos+1);
             }
-            
+
             ++pos;
         }
     }
diff --git a/src/package.h b/src/package.h
index bc3e055..ae5ae46 100644
--- a/src/package.h
+++ b/src/package.h
@@ -55,7 +55,7 @@ public:
     const bool hasReadme() const;
     const bool hasPreInstall() const;
     const bool hasPostInstall() const;
-    
+
     std::string versionReleaseString() const;
 
     void setDependencies( const std::string& dependencies );
@@ -103,7 +103,7 @@ struct PackageData
     bool hasReadme;
     bool hasPreInstall;
     bool hasPostInstall;
-    
+
     void generateVersionReleaseString();
 };
 
diff --git a/src/pg_regex.cpp b/src/pg_regex.cpp
index 0051beb..bf1995f 100644
--- a/src/pg_regex.cpp
+++ b/src/pg_regex.cpp
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2005 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 
@@ -16,25 +16,25 @@ using namespace std;
 #include "pg_regex.h"
 
 
-RegEx::RegEx(const string& pattern, bool caseSensitive) 
+RegEx::RegEx(const string& pattern, bool caseSensitive)
 {
     int additionalFlags = 0;
     if (!caseSensitive) {
         additionalFlags |= REG_ICASE;
     }
 
-    m_validPattern = 
-        regcomp(&m_pattern, 
-                pattern.c_str(), 
+    m_validPattern =
+        regcomp(&m_pattern,
+                pattern.c_str(),
                 REG_EXTENDED|REG_NOSUB|additionalFlags) == 0;
 }
 
-RegEx::~RegEx() 
+RegEx::~RegEx()
 {
     regfree(&m_pattern);
 }
 
-bool RegEx::match(const string& input) 
+bool RegEx::match(const string& input)
 {
     if (!m_validPattern) {
         return false;
@@ -45,7 +45,7 @@ bool RegEx::match(const string& input)
 
 bool RegEx::match(const string& pattern,
                   const string& input,
-                  bool caseSensitive) 
+                  bool caseSensitive)
 {
     RegEx re(pattern, caseSensitive);
     return re.match(input);
diff --git a/src/pg_regex.h b/src/pg_regex.h
index 7714017..518387f 100644
--- a/src/pg_regex.h
+++ b/src/pg_regex.h
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2005 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 
@@ -29,7 +29,7 @@ public:
     static bool match(const std::string& pattern,
                       const std::string& input,
                       bool caseSensitive=false);
-    
+
 private:
     regex_t m_pattern;
     bool m_validPattern;
diff --git a/src/pkgdb.cpp b/src/pkgdb.cpp
index c80c215..6fac548 100644
--- a/src/pkgdb.cpp
+++ b/src/pkgdb.cpp
@@ -55,7 +55,7 @@ bool PkgDB::isInstalled( const string& name,
     if ( !load() ) {
         return false;
     }
-    
+
     bool installed = m_packages.find( name ) != m_packages.end();
     if (!installed && useAlias) {
         string provider;
diff --git a/src/process.h b/src/process.h
index 416e65a..8a20a5e 100644
--- a/src/process.h
+++ b/src/process.h
@@ -21,7 +21,7 @@ using namespace std;
   \brief Process execution class
 
   A class to execute processes
-  
+
 */
 class Process
 {
@@ -31,13 +31,13 @@ public:
     int executeShell();
 
 private:
-    
+
     int exec(const int argc, char** argv);
     int execLog(const int argc, char** argv);
-    
+
     int execShell(const char* shell);
     int execShellLog(const char* shell);
-    
+
     string m_app;
     string m_arguments;
     int m_fdlog;
diff --git a/src/prtget.cpp b/src/prtget.cpp
index b03515e..9d057f6 100644
--- a/src/prtget.cpp
+++ b/src/prtget.cpp
@@ -544,7 +544,7 @@ void PrtGet::listInstalled()
 	// sort by dependency, without injecting missing ones
 	// calcDependencies chokes on the full list, so go through the
 	// ports one by one
-	
+
 	initRepo();
 	map<string, string>::iterator mit;
 	string name;
@@ -552,7 +552,7 @@ void PrtGet::listInstalled()
 	    mit = l.begin();
 	    name = mit->first;
 	    l.erase(mit);
-	
+
 	    InstallTransaction trans( name, m_repo, m_pkgDB, m_config );
 	    InstallTransaction::InstallResult result __attribute__((unused)) = trans.calcDependencies();
 	    const list<string>& depRef = trans.dependencies();
@@ -563,11 +563,11 @@ void PrtGet::listInstalled()
 		if (l.find(*it) != l.end()) {
 		    cout << *it << endl;
 		    l.erase(*it);
-		}	
+		}
 	    }
 	    cout << name << endl;
         }
-	
+
     } else {
 	for ( ; it != l.end(); ++it ) {
 	    if ( m_parser->verbose() > 1 ) {
@@ -1710,7 +1710,7 @@ void PrtGet::setLock( bool lock )
             }
         }
     }
-								
+
     if (!m_locker.store()) {
         cerr << "Failed to write lock data" << endl;
         m_returnValue = PG_GENERAL_ERROR;
@@ -2157,13 +2157,13 @@ void PrtGet::dumpConfig()
     cout.setf( ios::left, ios::adjustfield );
     cout.width( 20 );
     cout.fill( ' ' );
-    cout << "  Package dir: " << InstallTransaction::getPkgmkPackageDir() 
+    cout << "  Package dir: " << InstallTransaction::getPkgmkPackageDir()
          << endl;
-    
+
     cout.setf( ios::left, ios::adjustfield );
     cout.width( 20 );
     cout.fill( ' ' );
-    cout << "  Compression mode: " 
+    cout << "  Compression mode: "
          << InstallTransaction::getPkgmkCompressionMode() << endl;
 
 
diff --git a/src/signaldispatcher.cpp b/src/signaldispatcher.cpp
index 0c717a3..ec7322b 100644
--- a/src/signaldispatcher.cpp
+++ b/src/signaldispatcher.cpp
@@ -33,7 +33,7 @@ SignalDispatcher* SignalDispatcher::instance()
 
 void SignalDispatcher::dispatch( int signalNumber )
 {
-    map<int, SignalHandler*>::iterator it = 
+    map<int, SignalHandler*>::iterator it =
         SignalDispatcher::instance()->m_signalHandlers.find( signalNumber );
     if ( it !=  SignalDispatcher::instance()->m_signalHandlers.end() ) {
         it->second->handleSignal( signalNumber );
diff --git a/src/signaldispatcher.h b/src/signaldispatcher.h
index d29cdab..5b1f965 100644
--- a/src/signaldispatcher.h
+++ b/src/signaldispatcher.h
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2002 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 #ifndef _SIGNALDISPATCHER_H_
@@ -15,7 +15,7 @@
 #include <map>
 
 /*!
-  signal handler for the SignalDispatcher class. Implement this class to 
+  signal handler for the SignalDispatcher class. Implement this class to
   receive signals
   \brief SignalHandler for SignalDispatcher
 */
@@ -23,16 +23,16 @@ class SignalHandler
 {
 public:
     /*! Result of a handlSignal() call */
-    enum HandlerResult { 
+    enum HandlerResult {
         SIGNAL_NOT_HANDLED, /*!< not handled */
         EXIT,               /*!< signal handled, exit now */
         CONTINUE            /*!< signal handled, don't exit */
-    }; 
+    };
     virtual HandlerResult handleSignal( int signalNumber ) = 0;
 };
 
 /*!
-  dispatches signals. Singleton, use the instance() method to access 
+  dispatches signals. Singleton, use the instance() method to access
   the instance of this class. Register your SignalHandler to handle signals
 
   \brief Dispatch unix signals
@@ -42,17 +42,17 @@ class SignalDispatcher
 public:
     static SignalDispatcher* instance();
     static void dispatch( int signalNumber );
-    
+
     void registerHandler( SignalHandler* handler, int signalNumber );
-    void unregisterHandler( int signalNumber );    
-    
+    void unregisterHandler( int signalNumber );
+
 protected:
     SignalDispatcher();
-    
+
 private:
     static SignalDispatcher* m_instance;
     std::map<int, SignalHandler*> m_signalHandlers;
-    
+
 };
 
 #endif /* _SIGNALDISPATCHER_H_ */
diff --git a/src/stringhelper.cpp b/src/stringhelper.cpp
index 2b4f783..7ef5230 100644
--- a/src/stringhelper.cpp
+++ b/src/stringhelper.cpp
@@ -82,7 +82,7 @@ bool startsWith( const string& s, const string& with )
 {
     if (s.length() < with.length())
         return false;
- 
+
     return s.substr(0, with.length()) == with;
 }
 
diff --git a/src/versioncomparator.cpp b/src/versioncomparator.cpp
index b337ddf..49f75f4 100644
--- a/src/versioncomparator.cpp
+++ b/src/versioncomparator.cpp
@@ -251,7 +251,7 @@ int main(int argc, char** argv)
 
         check("7.0r63-3", "7.0r68-1", LESS);
         check("27", "28e", LESS);
-        
+
     } else {
         check(argv[1], argv[2], UNDEFINED, false);
     }
diff --git a/src/versioncomparator.h b/src/versioncomparator.h
index bdda33c..f84bd01 100644
--- a/src/versioncomparator.h
+++ b/src/versioncomparator.h
@@ -3,10 +3,10 @@
 // AUTHOR:      Johannes Winkelmann, jw@tks6.net
 // COPYRIGHT:   (c) 2004 by Johannes Winkelmann
 // ---------------------------------------------------------------------
-//  This program is free software; you can redistribute it and/or modify  
-//  it under the terms of the GNU General Public License as published by  
-//  the Free Software Foundation; either version 2 of the License, or     
-//  (at your option) any later version.                                   
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 2 of the License, or
+//  (at your option) any later version.
 ////////////////////////////////////////////////////////////////////////
 
 #ifndef _VERSIONCOMP_H_
-- 
2.39.1

