--- httpup-0.5.0/main.cpp.orig	2019-04-06 19:10:06.000000000 +0200
+++ httpup-0.5.0/main.cpp	2023-08-29 09:20:27.065825556 +0200
@@ -89,7 +89,7 @@
     } else {
         char* pwd = new char[256];
         if (getcwd(pwd, 256) == NULL) {
-            delete pwd;
+            delete [] pwd;
             pwd = new char[1024];
             if (getcwd(pwd, 1024) == NULL) {
                 cerr << "Path longer then 1024 characters; exiting" << endl;
